src/screens/poa-form/index.js

```diff
@@ -25,30 +25,32 @@ type AddressKeysType =
   | 'region'
   | 'postal_code'
 
-type PoAFormProps = {
+type PoAFormProps = {|
   t: string => string
-}
+|}
 
-type FormInputErrors = {
+type FormInputErrors = {|
   childrenNames: ?boolean,
   parentalStatus: ?boolean,
   parentalStatusReason: ?boolean,
   motherAddress: {[AddressKeysType]: ?boolean},
   fatherAddress: {[AddressKeysType]: ?boolean},
   caregiverAddress: {[AddressKeysType]: ?boolean}
-}
-type PoAFormState = {
+|}
+
+type PoAFormState = {|
   acceptedModal: boolean,
   step: number,
   numberOfChildren: number,
   submitted: boolean,
-  errors: FormInputErrors
-} & FormInputs
+  errors: FormInputErrors,
+  ...FormInputs
+|}
 
 class PoAForm extends Component<PoAFormProps, PoAFormState> {
   static navigationOptions = ({navigation}) => ({})
 
-  constructor(props) {
+  constructor(props: PoAFormProps) {
```

Comment:
This looks like it is also enforcing type.

Reply:
correct